cmd/agent/install_mac_os.sh

```diff
@@ -32,6 +36,26 @@ if [ -n "$DD_SITE" ]; then
     site=$DD_SITE
 fi
 
+systemdaemon_install=false
+systemdaemon_user_group=
+if [ -n "$DD_SYSTEMDAEMON_INSTALL" ]; then
+    systemdaemon_install=$DD_SYSTEMDAEMON_INSTALL
+    if [ -n "$DD_SYSTEMDAEMON_USER_GROUP" ]; then
+        systemdaemon_user_group=$DD_SYSTEMDAEMON_USER_GROUP
+    else
+        printf "\033[31mDD_SYSTEMDAEMON_INSTALL set without DD_SYSTEDAEMON_USER_GROUP\033[0m\n"
+        exit 1;
+    fi
+    if ! echo "$systemdaemon_user_group" | grep "^[^:]\+:[^:]\+$" > /dev/null; then
+        printf "\033[31mDD_SYSTEDAEMON_USER_GROUP must be in format UID:GID or UserName:GroupName\033[0m\n"
+        exit 1;
+    fi
+    if echo "$systemdaemon_user_group" | grep ">\|<" > /dev/null; then
+        printf "\033[31mDD_SYSTEDAEMON_USER_GROUP can't contain '>' or '<', because it will be used in XML file\033[0m\n"
```

Comment:
```suggestion
        printf "\033[31mDD_SYSTEMDAEMON_USER_GROUP must be in format UID:GID or UserName:GroupName\033[0m\n"
        exit 1;
    fi
    if echo "$systemdaemon_user_group" | grep ">\|<" > /dev/null; then
        printf "\033[31mDD_SYSTEMDAEMON_USER_GROUP can't contain '>' or '<', because it will be used in XML file\033[0m\n"
```